ckan/controllers/group.py

```diff
@@ -295,15 +305,17 @@ def pager_url(q=None, page=None):
                     facets[facet] = facet
 
             # Facet titles
-            for plugin in plugins.PluginImplementations(plugins.IFacets):
-                if self.group_type == 'organization':
+            from ckan.controllers.organization import OrganizationController
+            if isinstance(self, OrganizationController):
```

Comment:
No, I'm confusing cases where we're looking at a specific org/group vs this controller search method. I agree with your comment that this would be nicer as a method overridden in OrganizationController, though. Hiding an isinstance checks in here makes it hard to discover the special case.